apps/web-portal-client/lib/app/AppContainer.jsx

```diff
@@ -1,25 +1,35 @@
-import React from 'react'
-import {Switch, Route, Redirect} from 'react-router-dom'
+import React, {useEffect} from 'react'
+import {Switch, Route, Redirect, useHistory, useLocation} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+import {LoginPage} from '@web-portal-client/features/auth/containers/LoginPage'
```

Comment:
import {LoginPage} from '@web-portal-client/features/auth/containers'

Reply:
done

Reply:
import {LoginPage} from '@web-portal-client/features/auth/containers'  required index.mjs in containers